draft-ietf-dnsop-svcb-https.md

```diff
@@ -1227,6 +1227,66 @@ HTTPS RRs, all connections will be upgraded to HTTPS, and clients will
 use HTTP/3 if they can.  Parameters are "bound" to each server pool, so
 each server pool can have its own protocol, ECH configuration, etc.
 
+## Multi-CDN
+
+The HTTPS RR is intended to support HTTPS services operated by
+multiple independent entities, such as different Content Delivery
+Networks (CDNs) or different hosting providers.  This includes
+the case where a service is migrated from one operator to another,
+as well as the case where the service is multiplexed between
+multiple operators for performance, redundancy, etc.
+
+This example shows such a configuration, with www.customer.example
+having different DNS responses to different queries, either over time
+or due to logic within the authoritative DNS server:
+
+     $ORIGIN customer.example.  ; A Multi-CDN customer domain
+     www 900 IN CNAME cdn1.svc1.example.      ; For some subset of DNS responses
+     www 900 IN CNAME customer.svc2.example.  ; For the remaining DNS responses
+
+     ; The apex is also aliased to www to match its configuration
+     @ 7200 IN HTTPS 0 www
+     ; Non-HTTPS-aware clients use non-CDN IPs
+       		A    203.0.113.82
+		AAAA 2001:db8:203::2
+
+     ; Resolutions following the cdn1.svc1.example path use these records.
+     ; This CDN uses a different alternative service for HTTP/3.
+     $ORIGIN svc1.example.  ; domain for CDN 1
+     cdn1 300 IN HTTPS 1 h3pool.svc1.example. alpn=h3 echconfig="123..."
```

Comment:
A TTL of 300 on the HTTPS records seems low.

Reply:
updated